sunpy/map/tests/test_map_factory.py

```diff
@@ -141,17 +143,25 @@ def test_patterns(self):
         pair_map = sunpy.map.Map(data, header)
         assert isinstance(pair_map, sunpy.map.GenericMap)
 
-    def test_errors(self):
+    def test_errors(self, tmpdir):
         # If directory doesn't exist, make sure it's listed in the error msg
         nonexist_dir = 'nonexist'
         directory = pathlib.Path(filepath, nonexist_dir)
         with pytest.raises(ValueError, match=nonexist_dir):
-            maps = sunpy.map.Map(os.fspath(directory))
+            sunpy.map.Map(os.fspath(directory))
 
         with pytest.raises(ValueError, match='Invalid input: 78'):
             # Check a random unsupported type (int) fails
             sunpy.map.Map(78)
 
+        # If one file failed to load, make sure it's listed in warning msg
```

Comment:
This tests if an error is raised, not a warning.

Reply:
Fixed.